geometry.zsh

```diff
@@ -2,263 +2,40 @@
 # Based on Avit and Pure
 # avit: https://github.com/robbyrussell/oh-my-zsh/blob/master/themes/avit.zsh-theme
 # pure: https://github.com/sindresorhus/pure
-
-# Define how to colorize before the variables
-prompt_geometry_colorize() {
-  echo "%F{$1}$2%f"
-}
+GEOMETRY_ROOT=${0:A:h}
+source "$GEOMETRY_ROOT/lib/async.zsh"
+source "$GEOMETRY_ROOT/lib/plugin.zsh"
+source "$GEOMETRY_ROOT/lib/time.zsh"
+source "$GEOMETRY_ROOT/lib/color.zsh"
+source "$GEOMETRY_ROOT/lib/grep.zsh"
```

Comment:
On a build step. Maybe it's too cumbersome as of now. I'm gonna leave it as it is, sourcing them as currently implemented.